src/components/general/mediaTypes/ImagePopup.tsx

```diff
@@ -112,39 +132,46 @@ const ImagePopup = (props: Props) => {
                                 </Formik>
                             </div>
 
-                            <div className={classInfoBlock}>
-                                <Row>
-                                    <Col>
-                                        <p className="fs-4">
-                                            {(selectedAnnotation?.body && selectedAnnotation.body.length) ?
-                                                selectedAnnotation.body[0].value[0] : ''
-                                            }
-                                        </p>
-                                    </Col>
-                                    {(KeycloakService.IsLoggedIn() && selectedAnnotation?.body.length
-                                        && KeycloakService.GetSubject() === selectedAnnotation.body[0].creator.id) &&
-                                        <>
-                                            <Col className="col-md-auto">
-                                                <FontAwesomeIcon icon={faPencil}
-                                                    className="c-pointer"
-                                                    onClick={() => SetEditAnnotation(selectedAnnotation)}
-                                                />
-                                            </Col>
-                                            <Col className="col-md-auto ps-0">
-                                                <FontAwesomeIcon icon={faTrashCan}
-                                                    className="c-pointer"
-                                                    onClick={() => RemoveAnnotation()}
-                                                />
-                                            </Col>
-                                        </>
-                                    }
-                                </Row>
-                            </div>
+                            {annotation &&
```

Comment:
It wasn't possible to reuse the same component?

Reply:
Yeah let's do that, I will pick this up after the current stuff then